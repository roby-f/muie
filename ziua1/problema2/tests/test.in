2 20
kqb
iv
v
qdzuurpb
kwpu
tyrmoti
qu
z
avh
ss
ipa
pdxjvxobj
dicedyzsq
kthtrpfyz
aq
wqmncjgp
g
ff
upsjjh
rjsuukla
