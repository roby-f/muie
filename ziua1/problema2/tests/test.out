16
kqb
iv
qdzuurpb
kwpu
tyrmoti
avh
ss
ipa
pdxjvxobj
dicedyzsq
kthtrpfyz
aq
wqmncjgp
ff
upsjjh
rjsuukla
